### PR TITLE
test(client): install chrome-headless-shell for jest

### PR DESCRIPTION
**Pull request by jason-ha**
We use chrome-headless-shell; so, install that before testing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,9 +97,9 @@
 		"test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/nyc/**\" nyc",
 		"test:coverage": "c8 npm test",
 		"test:fromroot": "mocha \"packages/**/dist/test/**/*.spec.*js\" --exit",
-		"test:jest": "assign-test-ports && pnpm puppeteer browsers install chrome && pnpm -r --no-sort --stream --no-bail test:jest --color",
-		"test:jest:bail": "assign-test-ports && pnpm puppeteer browsers install chrome && pnpm -r --no-sort --stream test:jest",
-		"test:jest:report": "assign-test-ports && pnpm puppeteer browsers install chrome && pnpm -r --no-sort --stream --no-bail --workspace-concurrency=4 test:jest --ci",
+		"test:jest": "assign-test-ports && pnpm pnpm puppeteer browsers install chrome-headless-shell && pnpm -r --no-sort --stream --no-bail test:jest --color",
```
Comment (andre4i) [inline package.json:100]: extra `pnpm`?

Comment (jason-ha) [reply package.json:100]: too funny - indeed

Comment (jason-ha) [reply package.json:100]: I propose leaving it to get fix in and then I'll submit follow on.

Comment (jason-ha) [inline package.json:100]:
```suggestion
		"test:jest": "assign-test-ports && pnpm puppeteer browsers install chrome-headless-shell && pnpm -r --no-sort --stream --no-bail test:jest --color",
```

**Review by andre4i — APPROVED** [review]
🚀

**Pull request merged**